sycl/test/esimd/on-device/matrix_transpose_glb.cpp

```diff
@@ -10,6 +10,8 @@
 // REQUIRES: gpu
 // RUN: %clangxx-esimd -fsycl %s -o %t.out
 // RUN: %ESIMD_RUN_PLACEHOLDER %t.out
+// XFAIL: linux
```

Comment:
@bb-sycl, please, file a tracker for this failure. I don't think it's expected.
@kbobrovs, FYI.

Reply:
Thanks, @bader . Yes, this looks like a regression.

Reply:
@bader , I have file a tracker.